### DISCUSSION TITLE
Adding Costco to list of sites to monitor for PS5 drops

### DISCUSSION THREAD

**Discussion by Vuand** [jef/streetmerchant#1149] [Ideas and feature requests]
### Description

Is there add costco in the list of websites to monitor for ps5 drops. I'm having a lot of trouble making the .ts file for costco so any help would be greatly appreciated.

Thanks

**Comment by duff33** [comment]
I was able to create a working .ts file for Costco, however it required me to login to the site first.  When I open the site through an Incognito browser I can't use the cart selector to identify whether the membership item is in or out of stock.  I'm not sure if this automation is handling logins or if there is a way to cache the credentials when it runs for Costco.

**Comment by Vuand** [comment]
Yea, I'm having issues with that too. Does this program refresh tabs in an incognito window that causes sites to be logged out upon refresh?

**Comment by jef** [comment]
> Does this program refresh tabs in an incognito window

It can, but not by default (`INCOGNITO="true"`). As for logging in, _I believe_ you can load the chromium browser in the `node_modules` and that should save it's state.

It should be somewhere in this path: `node_modules/puppeteer/.local-chromium/`.
